_im-plugin/index-transforms/transforms-apis.md

```diff
@@ -125,35 +158,47 @@ PUT _plugins/_transform/<transform_id>
   }
 }
 ```
-You can specify the following options in the HTTP request body:
-
-Option | Data Type | Description | Required
-:--- | :--- | :--- | :---
-enabled | Boolean | If true, the transform job is enabled at creation. | No
-continuous | Boolean | Specifies whether the transform job should be continuous. Continuous jobs execute every time they are scheduled according to the `schedule` field and run based off of newly transformed buckets, as well as any new data added to source indexes. Non-continuous jobs execute only once. Default is false. | No
-schedule | Object | The schedule the transform job runs on. | Yes
-start_time | Integer | The Unix epoch time of the transform job's start time. | Yes
-description | String | Describes the transform job. | No
-metadata_id | String | Any metadata to be associated with the transform job. | No
-source_index | String | The source index whose data to transform. | Yes
-target_index | String | The target index the newly transformed data is added into. You can create a new index or update an existing one. | Yes
-data_selection_query | Object | The query DSL to use to filter a subset of the source index for the transform job. See [query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl) for more information. | Yes
-page_size | Integer | The number of buckets IM processes and indexes concurrently. Higher number means better performance but requires more memory. If your machine runs out of memory, IM automatically adjusts this field and retries until the operation succeeds. | Yes
-groups | Array | Specifies the grouping(s) to use in the transform job. Supported groups are `terms`, `histogram`, and `date_histogram`. For more information, see [Bucket Aggregations]({{site.url}}{{site.baseurl}}/opensearch/bucket-agg). | Yes if not using aggregations
-source_field | String | The field(s) to transform | Yes
-aggregations | Object | The aggregations to use in the transform job. Supported aggregations are: `sum`, `max`, `min`, `value_count`, `avg`, `scripted_metric`, and `percentiles`. For more information, see [Metric Aggregations]({{site.url}}{{site.baseurl}}/opensearch/metric-agg). | Yes if not using groups
 
 ## Update a transform job
 Introduced 1.0
 {: .label .label-purple }
 
-Updates a transform job if `transform_id` already exists.
+Updates the transform job if `transform_id` already exists. For this request you must specify the sequence number and primary term of the transform to be updated. To get these, use the [Get a transform job's details](#get-a-transform-jobs-details) API call. 
 
-**Sample Request**
+### Request format
 
 ```json
-PUT _plugins/_transform/<transform_id>
+PUT _plugins/_transform/<transform_id>?if_seq_no=<seq_no>&if_primary_term=<primary_term>
+```
+
+### Query parameters
 
+The Update operation supports the following URL parameters:
```

Comment:
Confirm that the capitalization of "Update" is intentional. Replace the colon with a period.